### ISSUE TITLE
Project data

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/eleventy-patterns#5]
It'd be neat to have a boilerplate `src/_data/project.json` file (or similar). Maybe it extends the `package.json` info (or provides access to it), but can allow configuration of page titles and other info like that?

**Comment by Paul-Hebert** [comment]
Yeah, that would be super neat! There should definitely be some sort
of easy config file along these lines.
